>From d000539173e8de112c804c47b9cea525d25be0f8 Mon Sep 17 00:00:00 2001
From: Salvatore Bonaccorso <carnil@debian.org>
Date: Sat, 17 Nov 2012 00:08:53 +0100
Subject: [PATCH] Improve init script stop and restart action

Use --name to killing an unrelated process. Furthermore use --retry to
wait for the process to actually die.

Don't sleep during restart action.
---
 debian/init |    3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/debian/init b/debian/init
index 8c24b29..d9d0121 100644
--- a/debian/init
+++ b/debian/init
@@ -41,14 +41,13 @@ case "$1" in
 
   stop)
 	echo -n "Stopping $DESC: "
-	start-stop-daemon --stop --oknodo --pidfile $PIDFILE
+	start-stop-daemon --stop --oknodo --pidfile $PIDFILE --name couriergraph.pl --retry 1
 	rm -f $PIDFILE
 	echo "$NAME."
   ;;
 
   restart|force-reload)
 	$0 stop
-	sleep 1
 	$0 start
   ;;
 
-- 
1.7.10.4

